### PR TITLE
e2e Test documentation for FE onboarding

### PR DESCRIPTION
**Pull request by calcaide**
## Description

This PR provides documentation to onboard FE devs to e2e testing:
- Prerequisites to run the test, accesses and software.
- Setup the tools need.
- Run enos (infrastructure) + Run e2e tests.

What we are missing in this PR (coming in future PR)
- Troubleshooting section: We want to document a troubleshooting guide with the common issues you can encounter working with Enos or writting test with Playwright.

### CODE REVIEW

File: ui/admin/tests/e2e/README.md [hunk 1]
```diff
@@ -1,36 +1,133 @@
...
 variety of environments as long as the appropriate environment variables are set. You can choose to
 spin up your own infrastructure or use [Enos](https://github.com/hashicorp/boundary/tree/main/enos)
-to generate it for you.
+to generate it for you. More information about [Enos available here.](https://github.com/hashicorp/Enos-Docs)
 
 The test suite uses the [Playwright](https://playwright.dev/) framework.
 
+## Prerequisites:
+
+You will need [Hombrebrew](https://brew.sh/) install. For secure persisting your SSH keys and tokens we recommend using 1Password.
```
Comment (moduli) [inline ui/admin/tests/e2e/README.md:34]:
```suggestion
You will need [Homebrew](https://brew.sh/) install. For secure persisting your SSH keys and tokens we recommend using 1Password.
```
though, homebrebrew sounds cool.

File: ui/admin/tests/e2e/README.md [hunk 2]
```diff
@@ -1,36 +1,133 @@
...
+
 ## Getting Started
-Setup...
-```bash
-cd boundary-ui/ui/admin
-yarn install
-npx playwright install # this installs the browsers used by Playwright
-```
+
+we will cover everything you need to setup before be able to run Enos + the tests.
```
Comment (moduli) [inline ui/admin/tests/e2e/README.md:54]:
```suggestion
We will cover everything you need to setup before be able to run Enos + the tests.
```

File: ui/admin/tests/e2e/README.md [hunk 3]
```diff
@@ -1,36 +1,133 @@
...
+
+### Setup Boundary CLI
```
Comment (moduli) [inline ui/admin/tests/e2e/README.md:56]:
```suggestion
### Set up Boundary CLI
```

File: ui/admin/tests/e2e/README.md [hunk 4]
```diff
@@ -1,36 +1,133 @@
...
-export E2E_PASSWORD_ADMIN_LOGIN_NAME=
-export E2E_PASSWORD_ADMIN_PASSWORD=
-export E2E_PASSWORD_AUTH_METHOD_ID=
-... # Tests may have additional variables that it needs
 cd boundary-ui/ui/admin
-yarn run e2e
+yarn install
+npx playwright install # this installs the browsers used by Playwright
 ```
+### Setup EC2 instance:
```
Comment (moduli) [inline ui/admin/tests/e2e/README.md:75]: To be clear, I don't think we're actually setting up an ec2 instance itself here

File: ui/admin/tests/e2e/README.md [hunk 5]
```diff
@@ -1,36 +1,133 @@
...
+
+**SSH Key pair:**
+
+You need to provide an SSH Key pair for the EC2 instance. We recommend creating specific e2e SSH keys and not re-use existing ones. To differentiate them, we recommend adding `enos` to the name, i.e: `name_enos`.
+
+[How to create EC2 SSH key pair](https://docs.aws.amazon.com/ground-station/latest/ug/create-ec2-ssh-key-pair.html).
+
+### Setup HCP Terraform (Terraform cloud):
+
+Login to [HCP Production env](https://portal.cloud.hashicorp.com/). Through HCP Terraform, open Terraform cloud, or [access here](https://app.terraform.io/). Then create an API token [following this documentation](https://developer.hashicorp.com/terraform/cloud-docs/users-teams-organizations/users#tokens).
```
Comment (moduli) [inline ui/admin/tests/e2e/README.md:87]:
```suggestion
Log in to [HCP Production env](https://portal.cloud.hashicorp.com/). Through HCP Terraform, open Terraform cloud, or [access here](https://app.terraform.io/). Then create an API token [following this documentation](https://developer.hashicorp.com/terraform/cloud-docs/users-teams-organizations/users#tokens).
```

File: ui/admin/tests/e2e/README.md (hunk 2 above)
Comment (lisbet-alvarez) [inline ui/admin/tests/e2e/README.md:54]:
```suggestion
We will cover everything you need to set up before being able to run Enos + the tests.
```

File: ui/admin/tests/e2e/README.md [hunk 6]
```diff
@@ -1,36 +1,133 @@
...
-export E2E_PASSWORD_ADMIN_PASSWORD=
-export E2E_PASSWORD_AUTH_METHOD_ID=
-... # Tests may have additional variables that it needs
 cd boundary-ui/ui/admin
-yarn run e2e
+yarn install
+npx playwright install # this installs the browsers used by Playwright
 ```
+### Setup EC2 instance:
+
```
Comment (lisbet-alvarez) [inline ui/admin/tests/e2e/README.md:76]: I'm wondering is this aws ec2 instance set up using the account requested through doormat (`boundary_team_acctest_dev`) ?

File: ui/admin/tests/e2e/README.md [hunk 7]
```diff
@@ -1,36 +1,133 @@
...
+You need to provide an SSH Key pair for the EC2 instance. We recommend creating specific e2e SSH keys and not re-use existing ones. To differentiate them, we recommend adding `enos` to the name, i.e: `name_enos`.
+
+[How to create EC2 SSH key pair](https://docs.aws.amazon.com/ground-station/latest/ug/create-ec2-ssh-key-pair.html).
+
+### Setup HCP Terraform (Terraform cloud):
+
+Login to [HCP Production env](https://portal.cloud.hashicorp.com/). Through HCP Terraform, open Terraform cloud, or [access here](https://app.terraform.io/). Then create an API token [following this documentation](https://developer.hashicorp.com/terraform/cloud-docs/users-teams-organizations/users#tokens).
+
+**Token awareness:** Copy your token from the box and save it in a secure location (we re). Terraform Cloud only displays the token once, right after you create it. And we will need the token later for enos configuration.
```
Comment (lisbet-alvarez) [inline ui/admin/tests/e2e/README.md:89]:
```suggestion
**Token awareness:** Copy your token from the box and save it in a secure location. Terraform Cloud only displays the token once, right after you create it. And we will need the token later for enos configuration.
```

File: ui/admin/tests/e2e/README.md [hunk 8]
```diff
@@ -1,36 +1,133 @@
...
+
+### Setup Enos:
+
+Enos needs some configuration variables to run the scenario successfully. [See the configuration file](https://github.com/hashicorp/boundary/blob/main/enos/enos.vars.hcl). The file has comments per each variable, but some awareness:
+
+- `aws_region`: The AWS region you are using. Very important as mentioned within the EC2 Setup.
+- `aws_ssh_keypair_name`: The name of the AWS keypair.
+- `aws_ssh_private_key_path`: The path to the private key associated with your keypair.
+- `enos_user`: the user name to use for Boundary.
```
Comment (lisbet-alvarez) [inline ui/admin/tests/e2e/README.md:98]:
```suggestion
- `enos_user`: The user name to use for Boundary.
```

**Review by lisbet-alvarez — APPROVED** [review]
Just left a couple suggestions. Amazing job with the docs!

File: ui/admin/tests/e2e/README.md (hunk 1 above)
Comment (calcaide) [reply ui/admin/tests/e2e/README.md:34]: :rofl: sorry to many brews. Thanks for the catch!

File: ui/admin/tests/e2e/README.md (hunk 7 above)
Comment (calcaide) [reply ui/admin/tests/e2e/README.md:89]: Nice catch Lis!!

**Review by randallmorey — APPROVED** [review]
Great work

**Pull request merged**